lib/layout/form.dart

```diff
@@ -158,6 +161,15 @@ class FormState extends WidgetState<EnsembleForm>
     for (Widget formItem in formItems) {
       if (formItem is HasController &&
           formItem.controller is FormFieldController) {
+        // Adding Form's readOnly value to the TextInput and PasswordInput
```

Comment:
This is wrong @vinothvino42 @snehmehta. The Form should have zero idea what a TextInput is, nor trying to reach inside its children and modify the child's data.

Reply:
Instead inside the child (i.g. TextInput) it should reach out to its parent Form and look at its attribute instead.

Reply:
Refactored it. Now I accessed the parent Form's readOnly property from the child widget (BaseTextInput's state class) and assigned the value to it if the BaseTextInput controller's readOnly property is null